pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/promtail/config.go

```diff
@@ -59,144 +71,21 @@ elif  [ ! -d "$TEMP_JOURNAL_FILE" ] && [ -d "$PERSISTANT_JOURNAL_FILE" ]; then
-type scrapeConfigs []job
-
-var defaultConfig = config{
-	Server: server{
-		Disable:        true,
-		LogLevel:       "info",
-		HTTPListenPort: ServerPort,
-	},
-	Client: client{
-		Url:             "http://localhost:3100/loki/api/v1/push",
-		BearerTokenFile: PathAuthToken,
-		TLSConfig: tlsConfig{
-			CAFile: PathCACert,
-		},
-	},
-	Positions: positions{
-		Filename: PositionFile,
-	},
-	ScrapeConfigs: scrapeConfigs{
-		{
-			"job_name": "journal",
-			"journal": map[string]interface{}{
-				"json":    false,
-				"max_age": "12h",
-				"path":    "/var/log/journal",
-				"labels": map[string]interface{}{
-					"job": "systemd-journal",
-				},
-			},
-			"relabel_configs": []map[string]interface{}{
-				{
-					"source_labels": []string{"__journal__hostname"},
-					"regex":         "^localhost$",
-					"action":        "drop",
-				},
-				{
-					"source_labels": []string{"__journal__systemd_unit"},
-					"target_label":  "unit",
-				},
-				{
-					"source_labels": []string{"__journal__hostname"},
-					"target_label":  "nodename",
-				},
-				{
-					"source_labels": []string{"__journal_syslog_identifier"},
-					"target_label":  "syslog_identifier",
-				},
-			},
-		},
-		{
-			"job_name": "containers",
-			"static_configs": []map[string]interface{}{
-				{
-					"targets": []interface{}{
-						"localhost",
-					},
-					"labels": map[string]interface{}{
-						"job":      "kube-system",
-						"nodename": "${HOSTNAME}",
-						"__path__": "/var/log/containers/*_kube-system_*log",
-					},
-				},
-			},
-			"pipeline_stages": []map[string]interface{}{
-				{
-					"cri": map[string]interface{}{},
-				},
-				{
-					"regex": map[string]interface{}{
-						"expression": "(?P<pod_name>[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)_(?P<namespace_name>[^_]+)_(?P<container_name>.+)-(?P<container_id>.+).log",
-						"source":     "filename",
-					},
-				},
-				{
-					"labels": map[string]interface{}{
-						"pod_name":       "",
-						"namespace_name": "",
-						"container_name": "",
-						"container_id":   "",
-					},
-				},
-				{
-					"labeldrop": []interface{}{
-						"filename",
-					},
-				},
-			},
-		},
-	},
-}
+func getPromtailConfigurationFile(ctx components.Context) (extensionsv1alpha1.File, error) {
+	var config bytes.Buffer
 
-func getPromtailConfiguration(ctx components.Context) (config, error) {
 	if ctx.LokiIngress == "" {
-		return config{}, fmt.Errorf("loki ingress url is missing for %s", ctx.ClusterDomain)
-	}
-	conf := defaultConfig
-	conf.Client.Url = "https://" + ctx.LokiIngress + "/loki/api/v1/push"
-	conf.Client.TLSConfig.ServerName = ctx.LokiIngress
-	return conf, nil
-}
-
-func getPromtailConfigurationFile(ctx components.Context) (extensionsv1alpha1.File, error) {
-	conf, err := getPromtailConfiguration(ctx)
-	if err != nil {
-		return extensionsv1alpha1.File{}, err
+		return extensionsv1alpha1.File{}, fmt.Errorf("loki ingress url is missing for %s", ctx.ApiServerURL)
```

Comment:
Shouldn't this be `ctx.LokiIngress` instead of `ctx.ApiServerURL` in the error message?

Reply:
If the `ctx.LokiIngress` then for which cluster the error is for?

Reply:
Ok, thanks.